src/inc/corinfo.h

```diff
@@ -663,7 +663,6 @@ enum CorInfoHelpFunc
     // Keep platform-specific helpers at the end so that the ids for the platform neutral helpers stay same accross platforms
     //
 
-#if defined(_TARGET_X86_) || defined(_HOST_X86_) // _HOST_X86_ is for altjit
                                     // NOGC_WRITE_BARRIERS JIT helper calls
```

Comment:
Even if you delete the #ifdefs, it seems like you should include some comments stating these are only used by X86 compilations. The comment immediately above this one also seems to now be obsolete.